### DISCUSSION TITLE
[nsfw] researchgate.net: badware

### DISCUSSION THREAD

**Discussion by MrToughFB** [uBlockOrigin/uAssets#26423] [Q&A]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.). Comment in the old issue threads even when they are closed.
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [X] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.researchgate.net/`

### Category

badware

### Description

It constantly opens this site, but I have never in my life accessed it before in my life, and I don't know whether it is genuine or not.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.61.2
Chromium: 131
filterset (summary):
 network: 141845
 cosmetic: 49376
 scriptlet: 23291
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 40912-141, 1h.54m Δ
  ublock-badware: 12339-1, 1h.54m Δ
  ublock-privacy: 1631-22, 1h.54m Δ
  ublock-unbreak: 2540-1, 1h.54m Δ
  ublock-quick-fixes: 223-0, 1h.54m Δ
  easylist: 78067-626, 1h.54m Δ
  easyprivacy: 53301-738, 1h.54m Δ
  urlhaus-1: 23630-0, 1h.54m
  plowe-0: 3541-0, now
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 172 ms (selfie)
 maxAssetCacheWait: 40 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 5
 network:
  amazon-adsystem.com: 1
  google-analytics.com: 1
  googletagmanager.com: 2
  pubmatic.com: 1
```
</details>

**Comment by stephenhawk8054** [comment]
It's a legit site. However, if the site popups everywhere, your computer might be infected with something. Scan your computer and check other extensions.

**Comment by MrToughFB** [comment]
Hi, so if this site is legit, then how do I fix my computer to make it stop popping up everywhere? What do you recommend me to do?

Thanks!

**Comment by ghost** [comment]
look in `chrome://extensions` -> any weird extension? consider disabling all other extensions, if any
you can take a look at https://www.malwarebytes.com/adwcleaner

**Comment by MrToughFB** [comment]
I have a Spotify adblocker, but i had downloaded that extension many months ago and other than that I had the Ublock origin and a Youtube ad blocker and some other ones related to work and school.
